docs/en/sql-reference/functions/other-functions.md

```diff
@@ -3873,39 +3873,7 @@ Retrieves the connection ID of the client that submitted the current query and r
 connectionId()
 ```
 
-**Parameters**
-
-None.
-
-**Returned value**
-
-Returns an integer of type UInt64.
-
-**Implementation details**
-
-This function is most useful in debugging scenarios or for internal purposes within the MySQL handler. It was created for compatibility with [MySQL's `CONNECTION_ID` function](https://dev.mysql.com/doc/refman/8.0/en/information-functions.html#function_connection-id) It is not typically used in production queries.
-
-**Example**
-
-Query:
-
-```sql
-SELECT connectionId();
-```
-
-```response
-0
-```
-
-## connection_id
-
-An alias of `connectionId`. Retrieves the connection ID of the client that submitted the current query and returns it as a UInt64 integer.
-
-**Syntax**
-
-```sql
-connection_id()
-```
+Alias: `connection_id`.
```

Comment:
In l. 3884, could we link `UInt64` to https://clickhouse.com/docs/en/sql-reference/data-types/int-uint?